stats_test.go

```diff
@@ -276,33 +276,31 @@ func TestPerInstanceStats(t *testing.T) {
 		},
 	}
 
-	sink := mock.NewSink()
-
-	testPerInstanceMethods := func(t *testing.T, scope Scope) {
+	testPerInstanceMethods := func(t *testing.T, setupScope func(Scope) Scope) {
```

Comment:
Can we add a test that explicitly checks that we flush zero valued counters? Could be super simple, I just wanna prevent regressions.